Source/storm/storm_svid.cpp

```diff
@@ -6,15 +6,19 @@
 
 #include <SmackerDecoder.h>
 
+#ifndef PS2
 #ifndef NOSOUND

```

Comment:
Consider doing this instead throughout:

```
#if !defined(NOSOUND) && !defined(PS2)
```

Reply:
Yeah, the svid portion is still just a the stage where i stubbed everything out to avoid linking with audiolib. I still have to find the best way to bring video sound back up.